enigma-principal/enclave/src/epoch_keeper_t/epoch_t.rs

```diff
@@ -25,32 +27,25 @@ impl Epoch {
             .get_selected_worker(sc_addr, self.seed)
             .ok_or_else(|| SystemError(EnclaveSystemError::WorkerAuthError { err: "Worker selection returns nothing.".to_string() }))
     }
-}
 
-impl RawEncodable for Epoch {
-    /// Encode the Epoch as Ethereum ABI parameters
-    fn raw_encode(&self) -> Bytes {
-        let raw_seed = H256::from(self.seed).0.to_vec();
-        let mut image = raw_seed.len().to_be_bytes().to_vec();
-        image.extend(raw_seed);
-
-        let raw_nonce = H256::from(self.nonce).0.to_vec();
-        image.extend(raw_nonce.len().to_be_bytes().to_vec());
-        image.extend(raw_nonce);
-
-        image.extend(self.worker_params.workers.len().to_be_bytes().to_vec());
-        for addr in self.worker_params.workers.clone() {
-            let raw_addr = addr.0.to_vec();
-            image.extend(raw_addr.len().to_be_bytes().to_vec());
-            image.extend(raw_addr);
-        }
-
-        image.extend(self.worker_params.stakes.len().to_be_bytes().to_vec());
-        for amount in self.worker_params.stakes.clone() {
-            let raw_amount = H256::from(amount).0.to_vec();
-            image.extend(raw_amount.len().to_be_bytes().to_vec());
-            image.extend(raw_amount);
-        }
-        image
+    pub fn encode_epoch(&self) -> Bytes {
```

Comment:
Very minor, but please change the name of `encode_epoch` to `encode` (no need for `epoch` twice). The name: `encode_for_hashing` or something alike even more preferable